third_party/sl4a/gradle.properties

```diff
@@ -1,6 +1,6 @@
 # This version code implements the versioning recommendations in:
 # https://blog.jayway.com/2015/03/11/automatic-versioncode-generation-in-android-gradle/
-VERSION_CODE=1040099
-VERSION_NAME=1.4.0
+VERSION_CODE=1040100
+VERSION_NAME=1.4.1-SNAPSHOT
```

Comment:
we shouldn't change this now, should we?

Reply:
Yeah, we should mark it to the next development iteration after released a new version.